From 3c2b9d96016b9a414490f834f9b1d4cd3fd39d85 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Thu, 21 Dec 2017 23:12:14 +0200
Subject: [PATCH 141/249] Replace hardware/cyanogen with
 hardware/lineage/lineagehw

* The original repository has been split to lineagehw
  and the light HALs that used to live there will be converted
  to HIDL HALs and go to hardware/lineage/interfaces

Change-Id: Idb4f74b3e0333a0c1aa6da476dab78b38bd3d2d2
---
 snippets/lineage.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index c295474..a67b1de 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -28,10 +28,10 @@
   <!--<project path="external/yaffs2" name="LineageOS/android_external_yaffs2" />-->
   <!--<project path="external/zip" name="LineageOS/android_external_zip" />-->
   <project path="frameworks/opt/hardware" name="LineageOS/android_frameworks_opt_hardware" />
-  <!--<project path="hardware/cyanogen" name="LineageOS/android_hardware_cyanogen" />-->
   <project path="hardware/lineage/interfaces" name="LineageOS/android_hardware_lineage_interfaces" >
     <copyfile dest="hardware/lineage/Android.bp" src="Android.bp"/>
   </project>
+  <project path="hardware/lineage/lineagehw" name="LineageOS/android_hardware_lineage_lineagehw" />
   <!--<project path="hardware/ti/omap4" name="LineageOS/android_hardware_ti_omap4" />-->
   <!--<project path="hardware/ti/wlan" name="LineageOS/android_hardware_ti_wlan" />-->
   <!--<project path="hardware/ti/wpan" name="LineageOS/android_hardware_ti_wpan" />-->
-- 
2.17.1

